Firestore/Protos/build-protos.sh

```diff
@@ -27,24 +27,38 @@ pod update
   --nanopb_out="--options-file=protos/%s.options:nanopb" \
   `find protos -name *.proto -print | xargs`
 
-# Remove "well-known" protos from objc. (We get these for free. We only need
-# them for nanopb.)
+# Remove "well-known" protos from objc and cpp. (We get these for free. We only
+# need them for nanopb.)
 rm -rf objc/google/protobuf/
+rm -rf cpp/google/protobuf
 
 # If a proto uses a field named 'delete', nanopb happily uses that in the
 # message definition. Works fine for C; not so much for C++. Rename uses of this
 # to delete_ (which is how protoc does it for c++ files.)
 perl -i -pe 's/\bdelete\b/delete_/g' `find nanopb -type f`
 
+# Rename nanopb's headers from foo.pb.h to foo.nanopb.h. This avoids collisions
```

Comment:
"This mostly doesn't matter ..." seems more flippant and verbose than it needs to be.

```sh
# Rename nanopb's headers from foo.pb.h to foo.nanopb.h. This avoids collisions
# with libprotobuf, which we use in tests to verify interoperation with our
# nanopb-based code.
```

Reply:
Was trying to convey that nanopb normally wants files named 'foo.pb.x'. i.e. the name collisions are "on-purpose" (ish) and that it's not really normal to want to use both. Maybe none of that matter though. Removed.